webpack.config.js

```diff
@@ -453,7 +476,46 @@ if (TARGET !== "readme") {
             maxEntrypointSize: prodMode ? 250000 /* the default */ : 10000000
         },
 
-        devtool: prodMode ? "hidden-source-map" : "source-map"
+        devtool: prodMode ? "hidden-source-map" : "source-map",
+
+        devServer: {
+            allowedHosts: "all", // Replace with docker parent and localhost
+            client: {
+                logging: "verbose",
+                overlay: {
+                    errors: true,
+                    warnings: false
+                },
+                progress: true
+            },
+            compress: true,
+            devMiddleware: {
+                index: false,
+                serverSideRender: true,
+                writeToDisk: true // Required for Perl TT
+            },
+            hot: true,
+            port: 9000,
+            proxy: {
```

Comment:
Very complete solution that it's also able to proxy! In this scenario we really don't need the nginx proxy, but I guess we'll just leave it in there for production simulation, right?

Reply:
Exact